jmclient/jmclient/blockchaininterface.py

```diff
@@ -668,7 +678,7 @@ def outputs_watcher(self, wallet_name, notifyaddr, tx_output_set,
         """
         wl = self.tx_watcher_loops[notifyaddr]
         account_name = wallet_name if wallet_name else "*"
-        txlist = self.rpc("listtransactions", [wallet_name, 100, 0, True])
+        txlist = self.rpc("listtransactions", ["*", 100, 0, True])
```

Comment:
What's the reason for using `"*"` for `listtransactions`? What happens when using the wallet name as before?

Reply:
That parameter is not supported anymore. The documentation says `1. "account"    (string, optional) DEPRECATED. The account name. Should be "*".`

Reply:
thanks for cluing me in